tests/unit_tests/test_client.py

```diff
@@ -682,7 +682,7 @@ def test_multiple_instances_no_warnings_or_errors_with_different_client_configs(
     UnleashClient(
         URL, "some-probably-unique-but-different-app-name", refresh_interval="60"
     )
-    assert not all(
+    assert not any(
```

Comment:
I think the change looks good but I'm not sure why these tests needed to change. Makes me a little nervous when we have to change unrelated tests and there's no new tests to cover the behaviour

Reply:
These tests are implicitly related to the change. 

Before it every time you create an UnleashClient without `scheduler` you get a warning log record. `not all(...)` was working, because the `msg` property of the record do not containe "Multiple instances has been disabled".  With this change there is no longer warning log records and `not all([])` becomes `False`. Logically the assertion should assert there isn't any log message about "Multiple instances has been disabled".

Reply:
Sure, makes sense! LGTM